sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -162,6 +170,9 @@ - (NSDictionary *)user {
             break;
     }
     userDict[@"gender"] = gender;
+	if ([[PrebidMobileDemandSDKLoaderSettings sharedInstance] isDemandEnabled:@"audienceNetwork"]) {
```

Comment:
I pass in buyeruid data no matter facebook is enabled or not, as long as it can be retrieved.

Reply:
What is the reasoning behind passing it no matter what?

I was thinking since we explicitly use the FB SDK to pull it in, we should only pass it in if Facebook demand is enabled.